container-host-files/etc/hcf/config/scripts/patches/fix_persi_nfsdriver.sh

```diff
@@ -0,0 +1,51 @@
+set -e
+
+PATCH_DIR="/var/vcap/jobs-src/nfsv3driver/templates"
+SENTINEL="${PATCH_DIR}/${0##*/}.sentinel"
+
+if [ -f "${SENTINEL}" ]; then
+  exit 0
+fi
+
+read -r -d '' ctl_patch <<'PATCH' || true
```

Comment:
I don't understand why we do this weird `read` business, instead of just
```bash
patch --force -d "${PATCH_DIR}" <<'PATCH'
@@ patch file goes here @@
PATCH
```

Reply:
consistency - the other scripts are the same;
iirc we tried this before and something didn't work - we could try again maybe? (add a card to refactor all the scripts)